src/fragments/start/getting-started/common/prereq.mdx

```diff
@@ -61,26 +59,26 @@ Once you're signed in, Amplify CLI will ask you to create an IAM user.
 
 ```console
 Specify the AWS Region
-? region:  # Your preferred region
+? region:  <Your preferred region>
 Specify the username of the new IAM user:
-? user name:  # User name for Amplify IAM user
-Complete the user creation using the AWS console
+? user name:  <User name for Amplify IAM user>
+Complete the user creation using the AWS console: <URL>
 ```
 
-Create a user with `AdministratorAccess-Amplify` to your account to provision AWS resources for you like AppSync, Cognito etc.
+Follow the `<URL>` and create a user with `AdministratorAccess-Amplify` to your account to provision AWS resources for you like AppSync, Cognito etc.
 
 ![image](/images/user-creation.gif)
 
-Once the user is created, Amplify CLI will ask you to provide the `accessKeyId` and the `secretAccessKey` to connect Amplify CLI with your newly created IAM user.
+Amplify CLI will then ask you to copy and past the `accessKeyId` and the `secretAccessKey` from your newly create IAM user to connect with Amplify CLI.
```

Comment:
Amplify CLI will then ask you to copy and paste the `accessKeyId` and the `secretAccessKey` from your newly created IAM user to connect with Amplify CLI.

Reply:
Endorsed